<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammingTestDialog</class>
 <widget class="QDialog" name="ProgrammingTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="consoleInputBrowseButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>78</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>108</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Console Output File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputBrowseButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>258</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add File...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>79</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Console Input File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="consoleInputEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>79</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>178</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Files:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>308</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Files:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="consoleOutputEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>108</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="consoleOutputBrowseButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>107</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="inputList">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>178</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="inputRemoveButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>258</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Remove File</string>
   </property>
  </widget>
  <widget class="QListWidget" name="outputList">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>308</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="outputBrowseButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>388</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add File...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputRemoveButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>388</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Remove File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Timeout (seconds):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeoutEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="consoleErrorEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>137</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>137</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Console Error File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consoleErrorBrowseButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>136</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>consoleInputBrowseButton</tabstop>
  <tabstop>consoleOutputBrowseButton</tabstop>
  <tabstop>consoleErrorBrowseButton</tabstop>
  <tabstop>inputBrowseButton</tabstop>
  <tabstop>outputBrowseButton</tabstop>
  <tabstop>timeoutEdit</tabstop>
  <tabstop>doneButton</tabstop>
  <tabstop>inputList</tabstop>
  <tabstop>outputList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
